src/pages/homepage/HomePage.Style.js

```diff
@@ -0,0 +1,88 @@
+import styled from "styled-components";
+
+// Purple color: #7030a0
+
+// Styled Components
+export const HeaderMain = styled.header`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 10rem auto 12rem;
+`;
+export const HeaderPage = styled.header`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 2rem auto;
+`;
+
+export const H1 = styled.h1`
```

Comment:
i suggest to give it another name than HTML tags , the naming is depending on the usage if its title -> Title , SubTitle->SubTitle